app/models/action/edition_diff_factory.rb

```diff
@@ -0,0 +1,45 @@
+class Action::EditionDiffFactory
+  attr_reader :previous_draft_edition, :updated_draft_edition
```

Comment:
Thinking more, it could be that these are public writer methods, and have private reader methods? not 100% sure on this though